test/service_instance_test/service_instance_test.go

```diff
@@ -65,8 +67,8 @@ var _ = test.DescribeTestsFor(test.TestCase{
 	ResourceType:                           types.ServiceInstanceType,
 	SupportsAsyncOperations:                true,
 	DisableTenantResources:                 true,
-	ResourceBlueprint:                      blueprint,
-	ResourceWithoutNullableFieldsBlueprint: blueprint,
+	ResourceBlueprint:                      blueprint(),
+	ResourceWithoutNullableFieldsBlueprint: blueprint(),
```

Comment:
This blueprint should return the object with only mandatory stuff set as far as i remember

Reply:
Yes, but there's no nullable properties that a client can ignore for instances, like there is for brokers and platforms (`description` property).